source/macos/AdaptiveCards/AdaptiveCards.xcodeproj/project.pbxproj

```diff
@@ -483,6 +483,7 @@
 		29430A5C25F216AA00CBA72E /* ContainerRenderer.swift in Sources */ = {isa = PBXBuildFile; fileRef = 29430A5B25F216AA00CBA72E /* ContainerRenderer.swift */; };
 		299DE17C26048FBB00605DAD /* ActionsRendererTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = 299DE17B26048FBB00605DAD /* ActionsRendererTests.swift */; };
 		29BA82D725E265F4000B7578 /* ChoiceSetInputRenderer.swift in Sources */ = {isa = PBXBuildFile; fileRef = 29BA82D625E265F4000B7578 /* ChoiceSetInputRenderer.swift */; };
+		29C3AD38260C7C6A00467069 /* UtiliOS.swift in Sources */ = {isa = PBXBuildFile; fileRef = 29C3AD37260C7C6A00467069 /* UtiliOS.swift */; };
 		29E8DF5225E69B2B000EC12F /* ACRChoiceButton.swift in Sources */ = {isa = PBXBuildFile; fileRef = 29E8DF5125E69B2B000EC12F /* ACRChoiceButton.swift */; };
```

Comment:
Guess `UtiliOS` is not used right?
This file can bee deleted.

Reply:
utilios has the ACRBleedDirection or should i merge it in baseCardelementRenderer